resources/js/old components/NewSynthVisuals2.js

```diff
@@ -0,0 +1,111 @@
+// import React from 'react';
+import Tone from 'tone';
+import pts, {
+    Pt,
+    Group,
+    Line,
+    Create,
+    Sound,
+    Triangle,
+    Const,
+    Geom
+} from 'pts';
+import { PtsCanvas } from "react-pts-canvas";
+import axios from 'axios';
+
+class NewSynthVisuals extends PtsCanvas {
+    constructor(props) {
+        super(props);
+        this.state = {
+            // Synth/param values
+            pitch: 400,
+            trigger: false,
+            pingPongFbk: 0,
+            chebWet: 0,
+            reverbWet: 0,
+            // Stores the synth and params in state
+            synth: null,
+            pong: null,
+            cheb: null,
+            reverb: null,
+        };
+        this.bins = 128; // range of: [32, 32768] --> 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384
+        this.minDB = -100; // default = -100
+        this.maxDB = -30; // d = -30
+        this.smooth = 0.8; // d = 0.8
+
+        this.synthTrigger = this.synthTrigger.bind(this);
+
+        // Sound.loadAsBuffer("https://upload.wikimedia.org/wikipedia/en/8/8f/Elvis_Presley_-_Are_You_Lonesome_Tonight.ogg").then(s => {
+        //     this.sound = s;
+        //     this.space.playOnce(50);
+        //     this.bufferLoaded = true;
+        // }).catch(e => console.error(e));
+        this.sound = null;
+    }
+
+    animate(time, ftime) {
+        if (this.sound) {
+            console.log(this.sound);
```

Comment:
Might want to be careful about leaving console.logs in. It can be tricky, especially if you're still using them for continued development.